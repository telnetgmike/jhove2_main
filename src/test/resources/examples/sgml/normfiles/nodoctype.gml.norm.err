/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/nodoctype.gml:2:0:1844095592.435:E: prolog can't be omitted unless CONCUR NO and LINK EXPLICIT NO and either IMPLYDEF ELEMENT YES or IMPLYDEF DOCTYPE YES
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/nodoctype.gml:2:0:1844095592.187:E: no document type declaration; will parse without validation
