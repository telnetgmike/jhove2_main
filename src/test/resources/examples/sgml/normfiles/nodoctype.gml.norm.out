<ART JID="jobojos" AID="1" VOL="1" ISSUE="1" FPAGE="1" LPAGE="1">

<COPYRIGHT MO="09" YR="1999">1999 by The Journal of Something.</COPYRIGHT>

<DOCHEAD>Journal Article</DOCHEAD>

<DOCTOPIC>Journal Articles</DOCTOPIC>



<FM>





<ATL>SOME TItle <SBT>By Allantoin Occurring in something</SBT></ATL>

<AUG>


<AU><FNM>John</FNM><SNM>OYBKUS</SNM></AU>






</AUG>









<ABS>

<P>LORE IPSE BOMUN.</P>

</ABS>



</FM></ART>
